src/contacthub.js

```diff
@@ -267,8 +267,17 @@ const allowedConfigOptions = [
   'token', 'workspaceId', 'nodeId', 'context', 'contextInfo'
 ];
 const config = (options: ConfigOptions): void => {
+  if (!(options.workspaceId && options.nodeId && options.token)) {
+    throw new Error('Invalid ContactHub configuration');
+  }
+
   // get current ch cookie, if any
-  const _ch = cookies.getJSON(cookieName) || {};
+  const currentCookie = cookies.getJSON(cookieName) || {};
+
+  // check if the auth token has changed
+  const hasTokenChanged = options.token && options.token !== currentCookie.token;
```

Comment:
here the check on `options.token` could be avoided (you're throwing if it's falsy)

Reply:
True, I added the `throw` afterwards and forgot to update this line!